src/CognitoClient.php

```diff
@@ -261,15 +323,27 @@ public function adminDeleteUser($username)
      * @param string $groupName
      * @throws Exception
      */
-    public function addUserToGroup($username, $groupName) {
+    public function addUserToGroup($username, $groupName)
+    {
         try {
             $this->client->adminAddUserToGroup([
                 'UserPoolId' => $this->userPoolId,
                 'Username' => $username,
                 "GroupName" => $groupName
             ]);
         } catch (CognitoIdentityProviderException $e) {
-            throw CognitoResponseException::createFromCognitoException($e);
+            if($e->getAwsErrorCode() == "ResourceNotFoundException"){
```

Comment:
As above, logic like this doesn't belong here. The calling code should create the group when it catches the `ResourceNotFoundException` exception.

Reply:
Corrected this